sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -217,31 +234,35 @@ public partial class ExtractKeyPhrasesResultCollection : System.Collections.Obje
         public string ModelVersion { get { throw null; } }
         public Azure.AI.TextAnalytics.TextDocumentBatchStatistics Statistics { get { throw null; } }
     }
-    public partial class HealthcareEntity : Azure.AI.TextAnalytics.Entity
```

Comment:
be specific please

Reply:
why you are no longer inheriting from Entity?

Reply:
This is how the .net api design says, https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7?diffRevisionId=da51140d3588434e8ea616dd8b1fdb1e&doc=False&diffOnly=False#Azure.AI.TextAnalytics.HealthcareEntity

@maririos

Reply:
@maririos I thought we talked in the arch board about this and we decided to make healthcare entity inherit from entity.

Reply:
hmm at least for now, .NET won't do it. If in the future we decide to, it is an additive change and not breaking so it wont be a problem.

We are not doing it because all of our entity types will need to be modified and currently we don't have the time. Prefer to be consistent than only having Healthcare inherit from Entity

Reply:
I see. I am curios though, why .Net entities need to be modified?

Reply:
so they can all inherit from `Entity` too and `HealthcareEntity` is not the only one.
We might consider doing this ins the future